scripts/host.sh

```diff
@@ -54,9 +54,12 @@ download_dependencies() {
     msg "copying genesis to configuration directory"
     cp "${shared_directory}/genesis.json" "${config_directory}/genesis.json"
 
-    msg "copying seeds to configuration directory"
+    msg "copying seeds.txt to configuration directory"
     cp "${shared_directory}/seeds.txt" "${config_directory}/seeds.txt"
```

Comment:
Isn't this file deleted now (but only for testnet)? We still need to copy this for mainnet.

Reply:
Oh yeah, we download the seeds from S3 which is still occurring. `add_seeds` is still being called in `download_dependencies`. Doesn't that clash with the new seeds in the config?

Reply:
I wouldn't say "clash". :P  The new axelar-core version will add seeds found in seeds.toml to whatever seeds it got from conventional sources (eg config.toml).  There might be duplicate seeds from those sources but I don't think it's a problem.

Reply:
Don't think we should be duplicating seeds. Scripts should either use `seeds.txt` or `seeds.toml`.

Reply:
Sure, but then we need to condition on the version inside the bash script.  Ick.

Good news!  axelar-core de-duplicates all seeds across all sources!
https://github.com/axelarnetwork/axelar-core/blob/a1560807ffdb6d921adad456318b40f3c95bf233/config/seeds.go#L39-L40

So there is no problem with duplicate seeds.